python/py-autobahn/Portfile

```diff
@@ -16,16 +16,23 @@ homepage            https://crossbar.io/autobahn
 master_sites        pypi:a/autobahn
 distname            autobahn-${version}
 
-checksums           rmd160  90a33d2f405a0581681cb0be4354507250d46202 \
-                    sha256  8fb9f3e7f6de5b7ee5479a56582873b927cc25d72319d0a86f43d992048d8171 \
-                    size    682775
+checksums           rmd160  d38811290f505c10cc941abd4cbd745e15f3136d \
+                    sha256  86bbce30cdd407137c57670993a8f9bfdfe3f8e994b889181d85e844d5aa8dfb \
+                    size    1260579
 
-python.versions     27 37 38
+python.versions     38
 
 if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
+    depends_lib-append \
```

Comment:
Are these also build or only runtime dependencies?

Reply:
These are runtime.  What's worse is that it seems to have a whole host more runtime deps depending on what features get turned on:

Listed under multiple `extra_requires_*` sections: https://github.com/crossbario/autobahn-python/blob/master/setup.py

Reply:
If they are not required at build time, it's better to use `depends_run-append`.

Reply:
OK, change made